index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html>
+    <head>
+        <title>base page</title>
+        <meta name="author" content="Sadat Rashid">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Mono:400,400i,700&amp;display=swap" rel="stylesheet">
+    </head>
+    <header>
+        <nav>
+            <a href>Home</a>
+            <a href> About me</a>
+            <a href class="active">Base page</a>
+        </nav>
+    </header>
+    <div class="main">
+        <section class="card">
+       <h2 class="card__tittle">karlplatz</h2>
+       <div class="card__imgwrapper"> 
+        <img class="card__img" src="karlpaltz_picture.jpeg">
+       </div>
+       <div class="card__description">
+           "   what I miss: no shop open at this time"
```

Comment:
why do you have double quotes around all your text? It is not needed unless maybe you are quoting something...